hoot-core/src/main/cpp/hoot/core/io/OsmPbfWriter.cpp

```diff
@@ -675,7 +681,6 @@ void OsmPbfWriter::_writePrimitiveBlock()
     int size = _d->primitiveBlock.ByteSize();
     _d->primitiveBlock.SerializePartialToArray(_getBuffer(size), size);
     _writeBlob(_buffer.data(), size, PBF_OSM_DATA);
-    _initBlob();
```

Comment:
While I was in here and making sure that `_writePrimitiveBlock()` didn't use the `_tick` member I found that the last line of `_writeBlob()` is a call to `_initBlob()` so the second one here isn't necessary.  So I removed the second call.